ecommerce/src/main/java/app/controller/SearchPageController.java

```diff
@@ -0,0 +1,27 @@
+package app.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+public class SearchPageController {
+	@Controller
+	public class SearchpageController {
+
+	    @RequestMapping("/ecommerce")
```

Comment:
ri sẽ hợp lý hơn nhé, hoặc để home
```suggestion
	    @RequestMapping("/")
```